arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp

```diff
@@ -237,32 +237,35 @@ void dropCollectionFromAllViews(
 ) {
   auto* vocbase = db.useDatabase(dbId);
```

Comment:
do you release the database later ?

Reply:
I haven't a clue, I didn't touch that code
I don't see any release in the original code either

Reply:
hm `useDatabase` should prevent anyone from dropping that db, if it works regardless of this use afterwards you are probably releasing it elsewhere